source/install/ee-install.rst

```diff
@@ -4,43 +4,46 @@
+* `Production Enterprise Edition on RHEL 6 <https://docs.mattermost.com/install/install-rhel-6.html>`__
+* `Production Enterprise Edition on Debian Stretch <https://docs.mattermost.com/install/install-debian.html>`__
+* `Production Docker Deployment using Docker Compose <https://docs.mattermost.com/install/prod-docker.html>`__
 
 Upgrading to Mattermost Enterprise Edition
 -------------------------------------------------
 
 Checking your Mattermost Edition and Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-If you're already running Mattermost, to check your Mattermost edition and version from the server run:
+If you're already running Mattermost, you can check Mattermost edition and version from the command line using:
 
   `mattermost version`
 
-To check your edition and version from the web interface, go to the main menu and open the "About Mattermost" dialog.
+To check your edition and version from the web interface, open **Main Menu > About Mattermost**.
 
-- "Mattermost Enterprise Edition" indicates your system is ready to receive a license key to enable enterprise features.
-- "Mattermost Team Edition" indicates you're using the open source version and need to upgrade (per upgrade instructions below) before a license key can be applied.
+- **Mattermost Enterprise Edition** indicates you can apply a license key to enable enterprise features.
+- **Mattermost Team Edition** indicates you're using the open source version and need to upgrade (per upgrade instructions below) before a license key can be applied.
 
 Upgrading to Enterprise Edition from Mattermost Team Edition
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-- The open source Mattermost Team Edition is functionally identical to the commercial Mattermost Enterprise Edition in its free "team mode", but there is no ability to unlock enterprise features.
-- To enable the unlocking of enterprise features, replace the Mattermost Team Edition binary with a Mattermost Enterprise Edition binary via an `upgrade procedure <http://docs.mattermost.com/administration/upgrade.html#upgrade-team-edition-to-enterprise-edition>`__.
+The open source Mattermost Team Edition is functionally identical to the commercial Mattermost Enterprise Edition in its free "team mode", but there is no ability to unlock enterprise features.
 
-If you need to migrate Team Edition prior to install, `please follow the migration guide. <http://docs.mattermost.com/administration/migrating.html>`__
+To enable unlocking enterprise features, replace the Mattermost Team Edition binary with a Mattermost Enterprise Edition binary via an `upgrade procedure <http://docs.mattermost.com/administration/upgrade.html#upgrade-team-edition-to-enterprise-edition>`__.
+
+If you need to migrate Team Edition prior to install, `please follow the migration guide <http://docs.mattermost.com/administration/migrating.html>`__.
```

Comment:
Maybe out of scope for this PR as I think this was part of the old doc - but what does "migrate Team Edition" mean here? It's not clear what kind of migration it's referring to

Reply:
Good question. I've reworded it slightly for more context.